core/data-plane/data-plane-framework/src/main/java/org/eclipse/dataspaceconnector/dataplane/framework/pipeline/PipelineServiceImpl.java

```diff
@@ -106,17 +115,20 @@ public CompletableFuture<StatusResult<Void>> transfer(DataSink sink, DataFlowReq
             return noSourceFactory(request);
         }
         var source = sourceFactory.createSource(request);
+        monitor.debug(() -> format("transferring from %s to %s.", source, sink));
         return sink.transfer(source);
     }
 
     @Override
     public void registerFactory(DataSourceFactory factory) {
         sourceFactories.add(factory);
+        monitor.info(() -> format("%s was registered: %s.", DataSourceFactory.class.getSimpleName(), factory.getClass().getSimpleName()));
     }
 
     @Override
     public void registerFactory(DataSinkFactory factory) {
         sinkFactories.add(factory);
+        monitor.info(() -> format("%s was registered: %s.", DataSinkFactory.class.getSimpleName(), factory.getClass().getSimpleName()));
```

Comment:
Please remove this statement as it is extraneous - the runtime will already emit a message that the extension was loaded, and it doesn't need to record a factory was registered.

Reply:
For example, I can not see the log about loading DataPlaneHttpExtension (which registers HttpDataSourceFactory and HttpDataSinkFactory) in the sample log like:

```
$ ./gradlew -p system-tests/e2e-transfer-test/runner clean test -DincludeTags="EndToEndTest" -PverboseTest --tests '*httpPushDataTransfer'
```

In addition, extensions can conditionally switch and register source/sink based on the condition. Logging which source/sink is registered is useful for finding issues even after the process is crashed. @jimmarino

Reply:
> For example, I can not see the log about loading DataPlaneHttpExtension

Sorry, it was not true.

```
    [provider-data-plane] INFO 2022-10-23T15:41:08.144923 Initialized Data Plane HTTP
```